src/mono/mono/benchmark/Makefile

```diff
@@ -0,0 +1 @@
+# empty placeholder for netcore build
```

Comment:
Why do we need the folder here at all?

Reply:
Yeah it's a quirk of how autotools works. We'll get rid of it once we turn off the mirror soon.